README_SYCL.md

```diff
@@ -40,6 +25,8 @@ The following CMake flags can be set for Intel CPUs:
 
 ### Building for Intel FPGAs
 
+Note: Intel FPGAs cannot be targeted together with other Intel hardware. This is because of different compiler trajectories as explained in the Intel oneAPI documentation.
```

Comment:
Could you add a link to the oneAPI documentation here please, thx!

Reply:
There is one at the bottom in the general notes :-)

Reply:
Fine.